src/schema/v2/profile.ts

```diff
@@ -63,17 +58,26 @@ export const ProfileType = new GraphQLObjectType<any, ResolverContext>({
         return followedProfileLoader(id).then(({ is_followed }) => is_followed)
       },
     },
+    isPubliclyVisible: {
+      type: GraphQLBoolean,
+      resolve: (profile) => profile && profile.published && !profile.private,
+    },
     isPublished: {
       type: GraphQLBoolean,
       resolve: ({ published }) => published,
     },
+    location: {
+      type: GraphQLString,
+      resolve: ({ location }) => location,
```

Comment:
This isn't a proper structured location?

Reply:
Not sure I follow

Reply:
Ah OK. Both user profiles and shows have locations that are more like this: https://github.com/artsy/metaphysics/blob/master/src/schema/v2/me/myLocation.ts